fuzz/api/main.go

```diff
@@ -14,9 +15,15 @@ func Fuzz(data []byte) int {
 		return -1
 	}
 
-	// should not panic
-	baker := agent.InitializeTemplateGenerator()
-	baker.GetNodeBootstrappingCmd(&config)
+	baker, err := agent.NewAgentBaker()
+	if err != nil {
+		return -2
+	}
+
+	_, err = baker.GetNodeBootstrapping(context.Background(), &config)
+	if err != nil {
+		return -3
+	}
 
-	return 1
+	return 0
```

Comment:
this should remain 1 to prioritize valid input as "interesting"

Reply:
oops